internal/util/util.go

```diff
@@ -71,27 +73,28 @@ func RequeueClusterOperator(c client.Client, name string) handler.MapFunc {
 	}
 }
 
-// InspectPlatformOperators iterates over all the POs on the cluster
+// InspectPlatformOperators iterates over all the POs in the list
 // and determines whether a PO is in a failing state by inspecting its status.
 // A nil return value indicates no errors were found with the POs provided.
-func InspectPlatformOperators(POList *platformv1alpha1.PlatformOperatorList) error {
-	var failingPOs []error
-	for _, po := range POList.Items {
+func InspectPlatformOperators(poList *platformv1alpha1.PlatformOperatorList) error {
+	var poErrors []error
+	for _, po := range poList.Items {
 		if err := inspectPlatformOperator(po); err != nil {
-			failingPOs = append(failingPOs, err)
+			poErrors = append(poErrors, err)
 		}
 	}
-	if len(failingPOs) > 0 {
-		return utilerror.NewAggregate(failingPOs)
-	}
-	return nil
+	return utilerror.NewAggregate(poErrors)
 }
 
 // inspectPlatformOperator is responsible for inspecting an individual platform
 // operator resource, and determining whether it's reporting any failing conditions.
 // In the case that the PO resource is expressing failing states, then an error
 // will be returned to reflect that.
 func inspectPlatformOperator(po platformv1alpha1.PlatformOperator) error {
+	if equality.Semantic.DeepEqual(po.Status, platformv1alpha1.PlatformOperatorStatus{}) &&
+		(po.CreationTimestamp.IsZero() || time.Since(po.CreationTimestamp.Time) < time.Minute) {
```

Comment:
Just want to highlight that my rebase included this addition based on the earlier discussion here: https://github.com/openshift/platform-operators/pull/40#discussion_r964424982